apps/src/StitchImages.cpp

```diff
@@ -0,0 +1,114 @@
+#include <iostream>
+#include <vector>
+
+#include <boost/filesystem.hpp>
+#include <boost/program_options.hpp>
+#include <opencv2/core.hpp>
+#include <opencv2/imgcodecs.hpp>
+
+#include "rt/ImageStitcher.hpp"
+
+
+namespace fs = boost::filesystem;
+namespace po = boost::program_options;
+
+auto divide_images = false;
+
+int main(int argc, char* argv[])
+{
+    // Adds the options that the user can set for the program
+    po::options_description stitching("Stitching Options");
+    stitching.add_options()
+            ("help,h", "Show this message")
+            ("images,imgs", po::value<std::vector<std::string> >()->multitoken()->required(), "Images to be stitched")
```

Comment:
If I take out the `multitoken()` call on the command line it would have to be `-i img1 -i img2 ...` but with that call it can just be `-i img1 img2 ...`. Is it ok to have the `-i` option for every image?

Reply:
Yeah, multiple -i is what I would expect